lib/beaker/options/command_line_parser.rb

```diff
@@ -94,6 +94,12 @@ def initialize
             @cmd_options[:debug_errors] = bool
           end
 
+          opts.on '--exec-manual-tests',
```

Comment:
In the current implementation of subcommands, this option will need to be copied over the `beaker test run` group.